backend/tests/Feature/CreateUserEndpointTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Tests\Feature;
+
+use Nuwave\Lighthouse\Testing\MakesGraphQLRequests;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+class CreateUserEndpointTest extends TestCase
+{
+    use MakesGraphQLRequests;
+    use RefreshDatabase;
+    /**
+     * A basic feature test example.
```

Comment:
This comment is too generic and the function name is already descriptive enough of what it does.
```suggestion

```

Reply:
Yep that's the default, thanks for catching it!

Reply:
Added.